### ISSUE TITLE
Transfer process state not moving beyond STARTED

### ISSUE THREAD

**Issue by sabarishrl** [eclipse-tractusx/tractusx-edc#833]
## Transfer process state not moving beyond STARTED
After getting contract agreement, start the transfer process. The state of the transfer process doesn't move beyond "Started". If there is any error it goes to "Terminated" but even after data is fetched the transfer status doesn't change to 'Completed'.

_Steps to reproduce the behavior:_

1. Get contract created using contract negotiation
2. Using contract negotiation start transfer process
3. Keep looking for state of the transfer process
4. Fetch the asset successfully
5. Check the transfer process status, it is showing as 'Started'

### Expected behavior

It should be in Completed or Success state

### Screenshots/Error Messages

No error

## Context Information

_Add any other context or information about the problem here._

- Used version: [e.g. Commit Hash]
- helm.sh/chart: tractusx-connector-0.5.1

**Comment by wolf4ood** [comment]
Hi @sabarishrl 

Thanks for raising this, which kind of transfer is it? `HttpProxy` ?

Thanks

**Comment by sabarishrl** [comment]
> Hi @sabarishrl
> 
> Thanks for raising this, which kind of transfer is it? `HttpProxy` ?
> 
> Thanks
Yes, tried with Http protocol that should be backed by http proxy

**Comment by ndr-brt** [comment]
This is expected, the `HttpProxy` is an ongoing transfer, so it will stay on `STARTED` until some external actors (like the `PolicyMonitor`, that's still on development) or an user set it to `TERMINATED`.
Anyway, after the token is expired, the transfer cannot happen anymore, also if it is `STARTED`.

**Comment by sabarishrl** [comment]
Thanks, If it is by user, using API we should update the state to Terminated?

**Comment by jerzy-wachowiak** [comment]
We observe the same behaviour EDC 0.5.1 Azure flavour. Log is flooded with: 
```
WARNING 2023-10-27T11:15:07.709049779 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:15:53.943236976 No checker found for process 249f6065-f78a-4c85-a64a-9ed9ee11a3af. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:16:25.974000402 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:17:08.462718459 No checker found for process 249f6065-f78a-4c85-a64a-9ed9ee11a3af. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:17:39.328585522 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:18:30.909160505 No checker found for process 249f6065-f78a-4c85-a64a-9ed9ee11a3af. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:18:45.599676176 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:19:32.466927772 No checker found for process 249f6065-f78a-4c85-a64a-9ed9ee11a3af. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:19:52.553583242 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:20:36.065419492 No checker found for process 249f6065-f78a-4c85-a64a-9ed9ee11a3af. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:21:09.490845823 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:21:54.807250595 No checker found for process 249f6065-f78a-4c85-a64a-9ed9ee11a3af. The process will not advance to the COMPLETED state.
WARNING 2023-10-27T11:22:08.381141973 No checker found for process 39302263-5149-4da0-895b-2bdb07d51b49. The process will not advance to the COMPLETED state.
```
Http Consumer-pull is used (https://github.com/eclipse-edc/Connector/blob/35b8c793095e8046fae2cd5487c9f5e5db786a28/docs/developer/architecture/data-transfer/diagrams/transfer-data-plane-consumer-pull.png)

**Comment by NSolutionDev** [comment]
Can I know which version this issue has been resolved?

**Comment by wolf4ood** [comment]
Hi @NSolutionDev  it should be resolved in 0.6.x releases

Thanks
